Backend/Remora.Discord.Caching/API/CachingDiscordRestChannelAPI.cs

```diff
@@ -679,5 +681,41 @@ CacheService cacheService
         await _cacheService.EvictAsync<IThreadMember>(key, ct);
 
         return result;
-    }
+    }
+
+    /// <inheritdoc />
+    public Task<Result<IChannelThreadQueryResponse>> ListPublicArchivedThreadsAsync

```

Comment:
These should either be implemented in the `Delegations.cs` part or perform some type of caching operation.